firebase-firestore/src/test/java/com/google/firebase/firestore/LoadBundleTaskTest.java

```diff
@@ -38,4 +58,208 @@ public void testImplementsAllTaskInterface() {
       }
     }
   }
+
+  @Test
+  public void testSuccessListener() throws InterruptedException {
```

Comment:
Please add tests to verify that the correct thread is used to perform the callbacks.

Reply:
Done